src/Settings/index.tsx

```diff
@@ -69,16 +70,21 @@ const MenuScreen: FunctionComponent = () => {
             lastItem
           />
         )}
+        <SettingsListItem
```

Comment:
Should this be conditionally rendered?
my understanding is that we dont want this on most of the builds.